class-00/lab-b/starter-code/app.js

```diff
@@ -156,45 +150,40 @@ let joe = new Student('Joe', 'Schmoe', 100);
 
 // TODO: Uncomment the following line of code to see the output in the browser console
 // Note that the arrow function will cause this code to break!
-// console.log(joe);
+console.log(joe);
 
 
-Student.prototype.greeting = function() {
-  return `Hi, my name is ${this.name}`;
-};
+Student.prototype.greeting = () => `Hi, my name is ${this.name}`;
 
 // TODO: Uncomment the following line of code to see the output in the browser console
 // Note that the arrow function will cause this method to break!
-// console.log(joe.greeting());
+console.log(joe.greeting());
 
 
-Student.courseName = function() {
-  return 'This student is enrolled in Code 301.';
-};
+Student.courseName = () => 'This student is enrolled in Code 301.';
+
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(Student.courseName());
+console.log(Student.courseName());
 
 
 
 // STEP 11
-// How do arrow functions affect constructor functions?
-Student.prototype.scope = function() {
-  console.log(this);
-};
+// How do arrow functions affect constructor functions? Arrow functions change the scope of this fromm block scope to global scope.
+Student.prototype.scope = () => console.log(this);
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scope());
+console.log(joe.scope());
 
 Student.prototype.scopeArrow = () => console.log(this);
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scopeArrow());
+console.log(joe.scopeArrow());
 
 // TODO: Write a COMMENT below to answer the following questions.
 // 1. What is "this" when joe.scope() is invoked?
-//
+// the Window object
 // 2. What is "this" when joe.scopeArrow() is invoked?
-//
+//  the Window object
 // 3. Explain why "this" is different when an arrow function is used.
-//
+// arrow functions scope this globally which breaks how they reference constructors

```

Comment:
Arrow functions don't scope tings globally, but the answer does have something to do with scoping